src/validation.js

```diff
@@ -220,39 +215,29 @@ export default class AuroFormValidation {
    */
   getErrorMessage(elem) {
     if (elem.validity !== 'valid') {
-      this.getAuroInputs(elem);
-
       if (elem.setCustomValidity) {
         elem.errorMessage = elem.setCustomValidity;
-      } else if (elem.tagName.toLowerCase() === 'auro-input') {
+      } else if (this.runtimeUtils.elementMatch('auro-input')) {
         const input = elem.renderRoot.querySelector('input');
 
         if (input.validationMessage.length > 0) {
           elem.errorMessage = input.validationMessage;
         }
-      } else if (this.auroInputElements && this.auroInputElements.length > 0) {
-        const firstInput = this.auroInputElements[0].renderRoot.querySelector('input');
+      } else if (this.inputElements && this.inputElements.length > 0) {
+        const firstInput = this.inputElements[0];
 
         if (firstInput.validationMessage.length > 0) {
           elem.errorMessage = firstInput.validationMessage;
-        } else if (this.auroInputElements.length === 2) {
-          const secondInput = this.auroInputElements[1].renderRoot.querySelector('input');
+        } else if (this.inputElements.length === 2) {
+          const secondInput = this.inputElements[1];
 
-          if (secondInput.validationMessage.length > 0) {
+          if (secondInput && secondInput.validationMessage.length > 0) {
```

Comment:
```suggestion
          if (secondInput.validationMessage.length > 0) {
```

I don't think we need that first check since this only runs when there is a second input.